src/read/elf/file.rs

```diff
@@ -61,6 +61,26 @@ impl<'data, Elf: FileHeader> ElfFile<'data, Elf> {
         })
     }
 
+    /// Returns the endianness.
+    pub fn endianess(&self) -> Elf::Endian {
+        self.endian
+    }
+
+    /// Returns the raw data.
+    pub fn data(&self) -> &Bytes<'data> {
+        &self.data
```

Comment:
```suggestion
    pub fn data(&self) -> Bytes<'data> {
        self.data
```

`Bytes` implements `Copy`, since it is a simple new-type for slice.